### PR TITLE
Create landing page

### PR DESCRIPTION
**Pull request by keelywan**
## Landing Page
Screenshot 2020-07-29 at 3 56 13 PM

Screenshot 2020-07-30 at 12 36 53 PM

Screenshot 2020-07-30 at 12 37 13 PM

## Footer
Screenshot 2020-07-30 at 12 37 43 PM

## Search Bar 
- Modified the search bar so it takes two props: `isDark` and `showSearchButton`
- This allows for the different appearance of the search bar on the landing page (passes `isDark="true"` and `showSearchButton="false"` to the component)

### CODE REVIEW

**Comment by andres-holguin** [comment]
Very nice! Feel free to allow the search-bar to come out as a dialog or something in this page, since based on #83 it can now route that information correctly to the search page.

**Review by Aradhya-B — APPROVED** [review]
Looks absolutely amazing!!! I made a few quick edits in #87 but feel free to reject any of those changes if you don't like them.

**Review by andres-holguin — APPROVED** [review]
Looks great, good eye for style!

File: src/main/webapp/components/Footer.vue [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+<style scoped>
+
+span {
+  padding: 3px;
+  display: inline-block;
+}
+
+img {
+  height: 20px;
```
Comment (nickKelman) [inline src/main/webapp/components/Footer.vue]: Nit: Still want these sorted? I don't view it as important but just a reminder.

**Review by nickKelman — APPROVED** [review]
This looks so good, amazing work.

**Review by cftorres — APPROVED** [review]
looking better and better!

File: src/main/webapp/components/Footer.vue (hunk 1 above)
Comment (keelywan) [reply src/main/webapp/components/Footer.vue]: Oops I missed that, thanks for catching it!

**Pull request merged**